code/modules/research/designs/misc_designs.dm

```diff
@@ -3,6 +3,24 @@
 /////////////////HUDs////////////////////
 /////////////////////////////////////////
 
+/datum/design/civ_hud
+	name = "Civilian HUD"
+	desc = "A heads-up display that shows only the current job ID of the user."
```

Comment:
this description is misleading and also worded really weirdly

Reply:
the other ones are too tbh (the items as well)

Reply:
I don't see how it's misleading, but if you have a better wording I'm happy to hear it.

Reply:
`current job ID of the user` but it doesn't do that, it shows you ids of everyone. also `only` doesnt make sense without context of the sechud.